test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/K8SequentialExecutor.java

```diff
@@ -147,9 +151,16 @@ private V1Pod getTestPod(String className, String methodName, String podName) {
                 ))
                 .restartPolicy("Never"));
         if (Utils.TLS_AND_AUTH_ENABLED) {
+            List<URI> ips = Futures.getAndHandleExceptions(client.getStatusOfPodWithLabel(NAMESPACE, "component", "pravega-controller")
```

Comment:
ips is not getting used, due to that no need to pass client to getTestPod function